lib/ui/text.dart

```diff
@@ -1828,9 +1828,10 @@ class ParagraphStyle {
                                           TextHeightBehavior._fromEncoded(_encoded[6], _leadingDistribution).toString() : "unspecified"}, '
              'fontFamily: ${    _encoded[0] & 0x080 == 0x080 ? _fontFamily                       : "unspecified"}, '
              'fontSize: ${      _encoded[0] & 0x100 == 0x100 ? _fontSize                         : "unspecified"}, '
-             'height: ${        _encoded[0] & 0x200 == 0x200 ? "${_height}x"                     : "unspecified"}, '
-             'ellipsis: ${      _encoded[0] & 0x400 == 0x400 ? '"$_ellipsis"'                    : "unspecified"}, '
-             'locale: ${        _encoded[0] & 0x800 == 0x800 ? _locale                           : "unspecified"}'
+             'strutStyle: ${    _encoded[0] & 0x200 == 0x200 ? _strutStyle                       : "unspecified"}, ',
```

Comment:
These bit masks must match the bits that are set in `_encodeParagraphStyle` (https://github.com/flutter/engine/blob/main/lib/ui/text.dart#L1671)

The order of the bits after `fontSize` is:
* `height`
* `strutStyle`
* `ellipsis`
* `locale`

Also, there should not be a comma after the closing quote of this string.  (The comma is causing the engine build failures in the CI checks)